src/components/RedirectExternal.tsx

```diff
@@ -6,18 +6,27 @@
  *
  */
 
-import { Component } from 'react';
-import PropTypes from 'prop-types';
-import { withRouter } from 'react-router-dom';
+import { Component, ContextType } from 'react';
+import { RouteComponentProps, withRouter } from 'react-router-dom';
+import { StaticContext } from 'react-router';
 /**
  * The react-router Redirect component does'nt work with external
  * urls. So we use this helper component to handle it for us.
  */
-class RedirectExternal extends Component {
-  constructor(props, context) {
+
+interface RedirectStaticContext extends StaticContext {
+  action: string;
+  location: string;
+  url: string;
+}
+interface Props extends RouteComponentProps<{}, RedirectStaticContext> {
+  to: string;
+}
+class RedirectExternal extends Component<Props> {
```

Comment:
Ser at denne kun brukes i ArticlePage for å redirekte til stier. ArticlePage brukes fra ResourcePage som på linje 92 sjekker om resource er læringssti og velger LearningpathPage. Det er mulig denne koden ikkje er aktiv i det heile tatt.

Reply:
Antar at denne vil bli slettet når vi går over til React Router 6. Tror det er bedre støtte for eksterne redirects der. Tror forsåvidt også `SafeLink` tar seg av eksterne redirects?